projects/angular/src/utils/i18n/common-strings.default.ts

```diff
@@ -73,10 +73,10 @@ export const commonStringsDefault: ClrCommonStrings = {
   // Stack View
   stackViewChanged: 'Value changed.',
   // Responsive Nav
-  responsiveNavToggleOpen: 'Navigation menu',
-  responsiveNavToggleClose: 'Navigation menu',
-  responsiveNavOverflowOpen: 'Navigation overflow menu',
-  responsiveNavOverflowClose: 'Navigation overflow menu',
+  responsiveNavToggleOpen: 'Open navigation menu',
+  responsiveNavToggleClose: 'Open navigation menu',
+  responsiveNavOverflowOpen: 'Open navigation overflow menu',
+  responsiveNavOverflowClose: 'Open navigation overflow menu',
```

Comment:
Shouldn't this be like that or?
```suggestion
  responsiveNavOverflowClose: 'Close navigation overflow menu',
```

Reply:
Thanks for catching this!